stdlib/public/core/Array.swift

```diff
@@ -1788,7 +1788,8 @@ extension Array {
   /// and enums.
   ///
   /// The following example copies bytes from the `byteValues` array into
-  /// `numbers`, an array of `Int`:
+  /// `numbers`, an array of `Int32`; on a [little-endian][1] machine, the
+  /// result is as shown:
```

Comment:
Because the big vs little endian information isn't the focus of the API being documented, what about moving this to a note box after the code listing?

```
The following example copies bytes from the `byteValues` array into
`numbers`, an array of `Int`:

[... code listing ...]

- Note: This example shows the behavior on a [little-endian][1] platform.
```

Reply:
Fixed as suggested